src/nerve/server/handlers/workflow_handler.py

```
@@ -0,0 +1,301 @@
+        # Start execution
+        await run.start()
+
+        # Optionally wait for completion
+        if params.get("wait", False):
+            try:
+                result = await run.wait()
+                return {
+                    "run_id": run.run_id,
+                    "state": run.state.value,
+                    "result": result,
+                }
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)